Golang tests are failing (due an upstream issue)

**Describe the bug**
The tests for `Go` are failing in the CI environment.

```
Run go get -u github.com/traefik/yaegi/cmd/yaegi
package io/fs: unrecognized import path "io/fs": import path does not begin with hostname
package github.com/traefik/yaegi/stdlib/syscall: build constraints exclude all Go files in /home/runner/go/src/github.com/traefik/yaegi/stdlib/syscall
package github.com/traefik/yaegi/stdlib/unsafe: build constraints exclude all Go files in /home/runner/go/src/github.com/traefik/yaegi/stdlib/unsafe
Error: Process completed with exit code 1.
```

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -140,7 +140,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
         matrix:
-            go-version: ["1.14", "1.13"]
+            go-version: ["1.17", "1.16"]
     steps:
       - uses: actions/checkout@v2
       - name: Setup python
@@ -149,8 +149,9 @@ jobs:
           python-version: "3.6"
       - name: Setup go
         uses: actions/setup-go@v2
-        #with:
-        #  go-version: ${{ matrix.go-version }}
+        with:
+          go-version: ${{ matrix.go-version }}
+      - run: go version
       - run: make deps-dev
       - run: go get -u github.com/traefik/yaegi/cmd/yaegi
       - run: PATH=$PATH:$(go env GOPATH)/bin  make lang-go-test

